### PR TITLE
Drop /jetty-spring/ module

### PR DESCRIPTION
**Pull request by joakime**
Signed-off-by: Joakim Erdfelt <joakim.erdfelt@gmail.com>

### CODE REVIEW

**Review by gregw — APPROVED** [review]
LGTM

**Review by sbordet — APPROVED** [review]

**Comment by joakime** [comment]
Waiting for green from CI before I merge.

**Review by olamy — APPROVED** [review]

**Pull request merged**
